recipes/default.rb

```diff
@@ -27,7 +27,11 @@
   end
 end
 
-require 'transmission-simple'
+ruby_block 'require transmission-simple gem' do
```

Comment:
I'm a bit confused by this change here. This is going to move the require into the converge phase and it's going to converge the node on every run by running this. For anyone doing reporting on changes they're going to see a change on every chef run even though nothing really happened.

Reply:
When the `require` is not within a `ruby_block` it will happen on a compile stageas, you say. However, that will require the `transmission-simple` gem to already exist.

When running this in a ChefSpec test, this means that you can't actually converge the test until you have installed a gem, which I'd say is a little counter productive, as unit tests shouldn't need to have any libraries like that installed, as external dependencies should be mocked out.

Reply:
However:

> For anyone doing reporting on changes they're going to see a change on every chef run even though nothing really happened.

Is very true. I'll see what options there are to avoid it marking that as "changed" on another Chef run.